api/core/tools/utils/yaml_utils.py

```diff
@@ -24,9 +25,6 @@ def load_yaml_file(file_path: str, ignore_error: bool = False) -> dict:
                 return yaml.safe_load(file)
             except Exception as e:
                 raise YAMLError(f'Failed to load YAML file {file_path}: {e}')
-    except FileNotFoundError as e:
```

Comment:
Can we now remove the `if` block at L20? Because `open` will throw this exception, it will be handled.

Reply:
Good suggestion. Adopted.